src/main/resources/schemas/schema-form.json

```diff
@@ -87,6 +97,12 @@
         "value"
       ]
     },
+    "ignoreExceptions": {
```

Comment:
I think it is better to rename this property to `ignoreTimeout` or `continueOnTimeout`